Ising_instruction.py

```diff
@@ -2,34 +2,81 @@
 # for the functions below, we need to write arguments, return variables and a brief description for each of them.
 # unfinished yet
 
-def initialize_lattice():
+def initialize_lattice(n): # the lattice will be represnted by a n x n matrix 
 ''' 
 The initialization of lattice should be the first step of simulation. In this function, 
 we give the lattice a random initial state by built-in function: np.random.random(), the
 
 '''
-    return
+    return np.random.choice([1, -1], size=(n, n))
 
 
-def spin_flip():
+def spin_flip(): # I dont think we need to do this in a seperate function 
```

Comment:
The flipping can be optimized for Wolff Algorithm, which flips a cluster of spins at a time. Maybe a separate function can make further optimization easier. But for now, add the spin_flip function into the main simulation function is okey.